extension/scripts/global/functions/filters.css

```diff
@@ -0,0 +1,90 @@
+.filter-container + * {
+	margin-top: 0 !important;
+}
+
+.filter-header {
+	margin-bottom: 10px;
+	display: flex;
+	flex-direction: row-reverse;
+}
+
+.filter-header .statistic span {
+	font-weight: 600;
+}
+
+.filter-content {
+	display: flex;
+	flex-wrap: wrap;
+}
+
+.tt-mobile .filter-content .filter-wrap {
+	flex: 1 0 100%;
+}
+
+.filter-wrap {
+	flex: 1 0 auto;
+	margin-bottom: 10px;
+}
+
+.filter-subwrap {
+	flex: 1 0 auto;
+	margin-bottom: 10px;
+}
+
+.filter-wrap .filter-heading {
+	text-align: center;
+	font-weight: 600;
+	margin-bottom: 5px;
+}
+
+.filter-slider {
+	width: 70%;
+	margin: auto auto 10px;
+}
+
+.filter-slider + .filter-slider-info {
+	width: 100%;
+	text-align: center;
+}
+
+.filter-wrap select {
+	width: 40%;
+	margin-left: 30%;
```

Comment:
I'm honestly not a fan of setting a margin to a %, as it mostly looks shit on other sizes.